lib/matplotlib/axes/_base.py

```diff
@@ -2764,10 +2764,10 @@ def ticklabel_format(self, *, axis='both', style='', scilimits=None,
           *scilimits*      (m, n), pair of integers; if *style*
                            is 'sci', scientific notation will
                            be used for numbers outside the range
-                           10`m`:sup: to 10`n`:sup:.
```

Comment:
Thanks for catching this.
I think
```
10\ :sup:`m`
```
(without the backslash-space on the other side) should be enough.

Reply:
You're right, according to the Sphinx documentation 

> Whitespace or punctuation is required around interpreted text

I changed the file accordingly, but again, I cannot easily test how it looks in the end...